lib/verdict/experiment.rb

```diff
@@ -21,6 +21,7 @@ def initialize(handle, options = {}, &block)
     @segmenter                   = options[:segmenter]
     @subject_type                = options[:subject_type]
     @disqualify_empty_identifier = options[:disqualify_empty_identifier]
+    @timestamps_out_of_band      = options[:timestamps_out_of_band]
```

Comment:
/nit for the sake of not adding another option here, could we just have a default implementation of `timestamps_out_band?` that returns `false` and have any experiment that wan't to use out of band timestamps just override it?